src/lib/nfv/selftest.sh

```diff
@@ -70,6 +70,9 @@ function stop_bench_env {
 
     # Give VMs and snabbnfv-traffic time to shut down.
     sleep 5
+
+    # Clean up temporary config location.
+    rm -f "$TESTCONFPATH"
```

Comment:
Can we do this on an [EXIT trap](http://redsymbol.net/articles/bash-exit-traps/) to make sure it happens even with `^C`? (Could be that this is how the whole `stop_bench_env` function should be invoked?)